### PR TITLE
Added param validation to didVisit

### PR DESCRIPTION
**Pull request by johnsmithtest1**
In reference to #3 

Currently the didVisit function in script.js is not being validated prior to executing its logic. Since this could lead to inaccurate results, I have added functionality that will verify a parameter has been provided. I would greatly appreciate it if you could review this pr and merge if acceptable. Thanks!

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -2,12 +2,15 @@ const visited = ["paris", "tokyo", "sydney", "new york"];
 
 
 //Verify if destination has been visited
-function didVisit(destination){
-    
-    if(visited.indexOf(destination) > -1){
+function didVisit(destination) {
+
+    if (!destination) {
```
Comment (zack-evans) [inline script.js:7]: Great job adding a check to verify that `destination` was provided!

File: script.js [hunk 2]
```diff
@@ -2,12 +2,15 @@ const visited = ["paris", "tokyo", "sydney", "new york"];
...
+        throw "Please provide a desitination as a parameter";
+    }
+
+    if (visited.indexOf(destination) > -1) {
         return true;
     }
+
     return false;
 }
-
```
Comment (zack-evans) [inline script.js:12]: Thank you for removing my `console.log`

**Review by zack-evans — COMMENTED** [review]
LGTM!

**Comment by johnsmithtest1** [comment]
@brntbeer could you please take a look at my pr?

**Review by brntbeer — COMMENTED** [review]
Looks good to me, but ill let @zack-evans have finals say!

Edit: typing on mobile is hard 😅

**Review by brntbeer — APPROVED** [review]

**Pull request merged**